File: client/AuditAdminClient/auditentity.cpp
#include "auditentity.h"

AuditEntity::AuditEntity()
{

}

int AuditEntity::getid()
{

	return id;

}
QDateTime AuditEntity::gettime()
{

	return time;

}
QString AuditEntity::getipaddress()
{

	return ipaddress;

}
QString AuditEntity::getusername()
{
	return username;

}
QString AuditEntity::getrole()
{
	return this->role;
}

QString AuditEntity::getmethod()
{
	return method;
}

QString AuditEntity::getmodule()
{
	return module;
}

QString AuditEntity::getobject()
{
	return object;
}

QString AuditEntity::getresult()
{
	return result;
}
QString AuditEntity::getstatus()
{
	return status;
}

void AuditEntity::setid(int id)
{
	this->id = id;
}

void AuditEntity::settime(QDateTime time)
{
	this->time = time;
}

void AuditEntity::setipaddress(QString ipaddress)
{
	this->ipaddress = ipaddress;
}
void AuditEntity::setusername(QString username)
{
	this->username = username;
}
void AuditEntity::setrole(QString role)
{
	this->role = role;
}
void AuditEntity::setmethod(QString method)
{
	this->method = method;
}
void AuditEntity::setmodule(QString module)
{
	this->module = module;
}
void AuditEntity::setobject(QString object)
{
	this->object = object;
}
void AuditEntity::setresult(QString result)
{
	this->result = result;
}
void AuditEntity::setstatus(QString status)
{
	this->status = status;
}

File: server/SecureCloudStorageSystem/src/main/java/com/sklois/SecureCloudStorageSystem/entity/AuditEntity.java
package com.sklois.SecureCloudStorageSystem.entity;


import jakarta.persistence.*;

import java.time.LocalDateTime;

@Entity //实体类注解
public class AuditEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;
    private LocalDateTime time; //时间
    private String ipaddress;//客户端IP地质
    private String username;//用户名
    private String role;//用户角色
    private String method;//操作方法,例如账户登录,账户退出,文件上传
    private String module;//模块,登录模块,文件浏览模块
    @Column(length = 2048)
    private String object;//访问资源,目标等,登录模块是xxxx端,文件上传是xxx文件
    private String result;//成功或者失败
    private String status;//状态,开始状态,执行状态,中断状态,异常状态,终止状态等.

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public LocalDateTime getTime() {
        return time;
    }

    public void setTime(LocalDateTime time) {
        this.time = time;
    }

    public String getIpaddress() {
        return ipaddress;
    }

    public void setIpaddress(String ipaddress) {
        this.ipaddress = ipaddress;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }

    public String getModule() {
        return module;
    }

    public void setModule(String module) {
        this.module = module;
    }

    public String getObject() {
        return object;
    }

    public void setObject(String object) {
        this.object = object;
    }

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}
